js/id7-popover.js

```diff
@@ -0,0 +1,106 @@
+import $ from 'jquery';
+
+/*
+ Customised Popover wrapper. Implements click away to dismiss.
+ */
+$.fn.id7Popover = function id7Popover(options) {
+  let $items = this;
+  const initClass = 'id7Popover-init';
+
+  // filter already initialized popovers
+  $items = $items.not(initClass);
+
+  const tooltipItems = $items.filter('.use-tooltip').toArray(); // boostrap js based tooltip
+
+  // set options, with defaults
+  const defaults = {
+    template: '<div class="popover"><div class="arrow"></div><div class="popover-inner"><button type="button" class="close" aria-hidden="true">&#215;</button><h3 class="popover-title"></h3><div class="popover-content"><p></p></div></div></div>',
```

Comment:
Super trivial, but as it's baked into ID7, should we use `fa-times...` rather than an entity here?

Reply:
Perhaps:

```
    template: '<div class="popover"><div class="arrow"></div><div class="popover-inner"><button type="button" class="close"><i class="fas fa-times" aria-hidden="true"></i><span class="sr-only">Close</span></button><h3 class="popover-title"></h3><div class="popover-content"><p></p></div></div></div>',
```